select_statements.sql

```diff
@@ -20,10 +20,45 @@ WHERE vehicle.stationName = "Uppsala station" AND booking.endDatum < "2024-04-13
 
 -- Underhållspersonal
 -- Sök fram alla bilar i behov av kontroll
+SELECT vehicle.registrationNumber FROM green_rental.vehicle
+LEFT JOIN booking_details ON vehicle.registrationNumber = booking_details.registrationNumber
+LEFT JOIN booking ON booking_details.bookingNumber = booking.bookingNumber
+LEFT JOIN control ON vehicle.registrationNumber=control.registrationNumber
+WHERE control.controlLarge is FALSE and control.datum < "2024-04-12" and booking.endDate = "2024-04-12";
```

Comment:
Predicate on controlLarge is not needed.

Reply:
Fixed